connector/manifest.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version='1.0' encoding='utf-8' ?>
 
-<connector-plugin class='mongosql_jdbc' superclass='jdbc' plugin-version='0.0.0' name='MongoSQL' version='18.1' min-version-tableau='2020.4'>
+<connector-plugin class='mongosql_jdbc' superclass='jdbc' plugin-version='0.1.0' name='MongoSQL' version='18.1' min-version-tableau='2021.1'>
```

Comment:
Let's talk with Alexi about this, but I don't _think_ we want to use "MongoSQL" here. Instead, I think "Atlas SQL" or "MongoDB" will be preferred, depending on the context

Reply:
For Tableau there aren't plans to provide a connector for ODBC.  
In the "Installed Connectors" section the connector names are generally the database name.  I do see some connectors in the "Additional Connectors" that show JDBC or ODBC, I'm not sure yet why there are the two sections.